src/components/Button/button.scss

```diff
@@ -1,73 +1,73 @@
 @import '../../scss/variables.scss';
 
-.styled-button {
-  position: relative;
-  display: inline-flex;
-  align-items: center;
-  justify-content: center;
-  vertical-align: top;
+// .styled-button {
```

Comment:
The reference to the deleted `variables.scss` above needs to be deleted to be able to run the app